stable/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 2.4.1
+version: 2.4.2
```

Comment:
Could you bump minor version instead of patch, since this is a new feature?

Reply:
No problem